boringtun/src/device/mod.rs

```diff
@@ -303,12 +303,12 @@ impl DeviceHandle {
         }
     }
 
-    fn new_thread_local(thread_id: usize, device_lock: &LockReadGuard<Device>) -> ThreadData {
+    fn new_thread_local(_thread_id: usize, device_lock: &LockReadGuard<Device>) -> ThreadData {
```

Comment:
Is the prefixing with `_` intentional here?

Reply:
Yes, because otherwise clippy complains that this variable is unused on mac os. Tho I'm wasn't sure if this is the best way to deal with this situation